tests/c_languages/CPPLintBearTest.py

```diff
@@ -26,3 +26,11 @@
     settings={'cpplint_ignore': 'legal',
               'max_line_length': '13'},
     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPLintBearInfiniteLineLengthTest = verify_local_bear(
+    CPPLintBear,
+    valid_files=(test_file,),
```

Comment:
you should test a file with a very long line-length. You can maybe generate one dynamically here to really stress-test it :+1:

Reply:
generating dynamically means?